engines/helm/app/models/k8s/cli.rb

```diff
@@ -94,12 +94,19 @@ def create_namespace(name)
       return stdout
     end
 
-    def delete_namespace(name)
-      args = %W(delete namespaces #{name})
-      stdout, stderr = execute(*args)
-      return stderr if stderr.present?
-      return stdout
-    end
+    # def delete_namespace(name)
```

Comment:
What's this for?

Reply:
when cleaning resources, there are things that are created inside the cluster/node, for some of that, deleting the node is enough. That method calls were commented out before, I commented the method itself to make sure it is not being called.

Reply:
If it's commented out can it be cleaned up?